src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.CompilationTracker.CompilationTrackerState.cs

```diff
@@ -164,20 +167,20 @@ private sealed class FinalCompilationTrackerState : CompilationTrackerState
             public override Compilation CompilationWithoutGeneratedDocuments { get; }
 
             private FinalCompilationTrackerState(
-                bool isFrozen,
+                CreationPolicy creationPolicy,
                 Compilation finalCompilationWithGeneratedDocuments,
                 Compilation compilationWithoutGeneratedDocuments,
                 bool hasSuccessfullyLoaded,
                 CompilationTrackerGeneratorInfo generatorInfo,
                 UnrootedSymbolSet unrootedSymbolSet)
-                : base(isFrozen, generatorInfo)
+                : base(creationPolicy, generatorInfo)
             {
                 Contract.ThrowIfNull(finalCompilationWithGeneratedDocuments);
 
                 // As a policy, all partial-state projects are said to have incomplete references, since the
                 // state has no guarantees.
                 this.CompilationWithoutGeneratedDocuments = compilationWithoutGeneratedDocuments;
-                HasSuccessfullyLoaded = hasSuccessfullyLoaded && !isFrozen;
```

Comment:
>&& !isFrozen; [](http://example.com/codeflow?start=62&length=13)

Just wanted to make sure this was intentional

Reply:
yes it was.  deciding to freeze things doesn't have any impact on our determination if things successfully loaded or not.